proto/control-api/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ All user visible changes to this project will be documented in this file. This p
 
 
 
+## [0.8.0] · 2022-??-?? · To-be-done
+[0.8.0]: /../../tree/medea-control-api-proto-0.8.0/proto/control-api
+
+[Diff](/../../compare/medea-control-api-proto-0.7.0...medea-control-api-proto-0.8.0)
+
+- Define `ControlApi` and `CallbackClient` traits ([#55]).
```

Comment:
I've also expected 4 traits here instead of 2:
- for client side (behind `"client"` feature): `CallbackServer` and `ControlApiClient`
- for media server side (behind `"server"` feature): `CallbackClient` and `ControlApiServer`

The trait semantics are quite different on server and client side, and I don't see a reasonable way they fit into a single trait.

Reply:
@tyranron how semantics of `ControlApiClient` and `ControlApiServer` are actually different? The way I see it, we can provide single `ControlApi`, where we have 2 implementations: one on `Medea` side and another one for protocol, `gRPC` for example. So on our backend (client), we just would have `grpc::control_api_client::ControlApiClient: ControlApi`, so we can just execute `client.create(...).await`. While on the `Medea` side we will provide `MedeaControl: ControlApi` and `gRPC` integration will have blanket impl `impl<T: ControlApi> grpc::control_api_server::ControlApi for T { ... }`, so we can just create `gRPC` server with `grpc::control_api_server::ControlApiServer::new(MedeaControl)`. 

The main idea of a single trait is that protocol layer should just pass `ControlApi` calls to actual `MedeaControl: ControlApi`, I don't see where we need 2 traits for that.

Reply:
@ilslv I guess that the implementation semantics may differ? But that's OK, I may be wrong. Let's try with a single trait first then.